pandas/io/excel.py

```diff
@@ -154,15 +156,17 @@
     argument to indicate comments in the input file. Any data between the
     comment string and the end of the current line is ignored.
 skip_footer : int, default 0
+    Rows at the end to skip (0-indexed).
 
     .. deprecated:: 0.23.0
+
        Pass in `skipfooter` instead.
 skipfooter : int, default 0
-    Rows at the end to skip (0-indexed)
-convert_float : boolean, default True
-    convert integral floats to int (i.e., 1.0 --> 1). If False, all numeric
+    Rows at the end to skip (0-indexed).
+convert_float : bool, default True
+    Convert integral floats to int (i.e., 1.0 --> 1). If False, all numeric
     data will be read in as floats: Excel stores all numbers as floats
-    internally
+    internally.
 
 Returns
 -------
```

Comment:
In the next line (github doesn't let me add a comment there, as it's not modified): `DataFrame or dict of DataFrame`.

Then, in the examples, I'd remove the part that creates the Excel files. I'm sure the users will understand that they need to have as parameter a file that exists.

Just add after the lines with `read_excel`: `# doctest: +SKIP` so it doesn't run in the doctests (and it doesn't fail).